### PR TITLE
WSTEAMA-364:  Item list schema format for Topic pages

### PR DESCRIPTION
**Pull request by karinathomasbbc**
Resolves JIRA https://jira.dev.bbc.co.uk/browse/WSTEAMA-364

Overall changes
======
Topic pages use the item list schema format (similar to new HomePage)

Code changes
======
- Create SEOUtils package and refactor reusable logic from homepage
- Use SEOUtils on HomePage to convert curation data into the correct linked data
- Adding a unit test to the Topic Page for SEO
- Use SEOUtils on TopicPage to convert curation data into the correct linked data

Testing
======
- [ ] Unit tests for new SEOUtils functions
- [ ] Existing HomePage + TopicPage unit tests still passing
- [ ] Manual - check home page and topic page rendering as expected
- [ ] Structured Data Tooling: ensure that homepage and topic page has expected structured data

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: src/app/lib/seoUtils/getListItems/index.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { CurationData } from '../../../models/types/curationData';
+
+export default (curations: CurationData[]) => {
```
Comment (amoore108) [inline src/app/lib/seoUtils/getListItems/index.ts]: Do `getListItems` and `getItemList` need to be separate things? Can the one function not just return the data needed? Just confused me a little with the naming, and I can't see them being used independently of one another.

Comment (karinathomasbbc) [reply src/app/lib/seoUtils/getListItems/index.ts]: Good shout, they technically could be reused later, but this is perhaps premature optimisation. I will combine them into a single function.

**Review by amoore108 — APPROVED** [review]
Looks great! 🚀

**Review by pvaliani — APPROVED** [review]
🚀

**Pull request merged**